stdlib/3/_ast.pyi

```diff
@@ -0,0 +1,358 @@
+# Python 3.5 _ast
+import typing
+from typing import Any, Optional, Union
+
+PyCF_ONLY_AST = ... # type: int
+
+identifier = str
+
+class AST:
+    _attributes = ... # type: typing.Tuple[str, ...]
+    _fields = ... # type: typing.Tuple[str, ...]
+    def __init__(self, *args, **kwargs) -> None: ...
```

Comment:
I guess you can add that (from https://docs.python.org/3/library/ast.html#ast.AST.col_offset), as `NameConstant` can pass `Any` as value. Also in stdlib/2/_ast.pyi.
```python
    def __init__(self, *args: Any, **kwargs: Any) -> None: ...
```